RCode/NEFI/partition_uncertainty_DayLengthQuad.R

```diff
@@ -0,0 +1,274 @@
+# uncertainty partitioning 
+
+
+# State-space model - revised by S. LaDeau (11/2017) from the EcoForecast Activity by Michael Dietze, with reference "Ecological Forecasting", chapter 8
+# ========================================================
+#   
+#   The data used for this example are from summer weekly(ish) Gloetrichia echinulata (Gloeo.) sampling at 4 locations in Lake Sunapee, NH. The data are provided by Kathryn Cottingham, and should not be used without permission outside this workshop.
+
+library(tidyverse)
+library(readxl)
+library(rjags)
+library(runjags)
+library(moments)
+library(geosphere)
+library(ecoforecastR)
+library(googledrive)
+source('RCode/helper_functions/google_drive_functions.R')
+source('RCode/NEFI/get_data.R')
+source('RCode/helper_functions/plug_n_play_functions.R')
+
+
+#1) Model options => pick date range, site, time step, and type of model -----------------------------------------------------
+
+start_date = '2007-01-01' # in YYYY-MM-DD format; 1st 
+end_date = '2015-12-31' # Excluding 2016-2017 to use as sample data
```

Comment:
How does changing the `end_date` change the relative uncertainty plots? Some of them aren't too exciting and I wonder if it's because the Gloeo abundance is so low during the winter that initial condition and/or parameter uncertainty don't matter very much. Maybe a quick check could be to create a plot using `2015-12-31` as an `end_date` and then a date during a "bloom"

Reply:
good idea! as written (if-else statement), we can't extend the prediction beyond October 2016 right now because that's the last point we have in the dataset, but I like the idea of ending at different times before that :)